core/config/config_test.go

```diff
@@ -67,3 +68,12 @@ func TestOverrideFromSystemEnv(t *testing.T) {
 		})
 	}
 }
+
+func TestGetConfig(t *testing.T){
+	SetConfig("test","v1")
```

Comment:
We now use yaml as config file.
It's better to add a case to verify whether sentinel could deserialize yaml file to Entity struct.

Reply:
done